tests/acceptance/features/webUIFiles/copyCurrentPath.feature

```diff
@@ -0,0 +1,13 @@
+Feature: copy current path as a permanent link
+  As a user
+  I want to copy the permanent link to the current folder
+  So that I can paste paste it else where
```

Comment:
is your goal to paste the link or actually to be able to send it to someone?

Reply:
yep, the intended usage is to share the link via messengers, email, etc

Reply:
changed to `So that I can share it with other users`